autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/SectionSelect/SectionInfo.tsx

```diff
@@ -49,7 +49,7 @@ const SectionInfo: React.FC<SectionInfoProps> = ({
   // show section number and remaining seats if this is the first meeting for a section
   const sectionHeader = (
     <Typography className={styles.denseListItem} component="div" style={{ display: 'flex', gridColumn: '1 / -1' }}>
-      <div style={{ flex: 1, display: 'flex' }}>
+      <div style={{ flex: 1, display: 'flex', alignItems: 'center' }}>
```

Comment:
Is there a reason we're doing styles in `.tsx` instead of `.css`? Just don't want to make it a class?

Reply:
Fixed